PendingReleaseNotes.md

```diff
@@ -2,10 +2,11 @@
 
 ## Breaking Changes
 
-*  The mds liveness and startup probes are now configured by the filesystem CR instead of the cluster CR. To apply the mds probes, they need to be specified in the filesystem CR. See the [filesystem CR doc](Documentation/ceph-filesystem-crd.md#metadata-server-settings) for more details. 
-Pr: https://github.com/rook/rook/pull/9550
+* The mds liveness and startup probes are now configured by the filesystem CR instead of the cluster CR. To apply the mds probes, they need to be specified in the filesystem CR. See the [filesystem CR doc](Documentation/ceph-filesystem-crd.md#metadata-server-settings) for more details. See #9550
 
 ## Features
 
-*  Add support for custom ceph.conf for csi pods.
-Pr: https://github.com/rook/rook/pull/9567
+* The default number of mgr daemons for new clusters is increased to 2 from 1, resulting in a standby mgr daemon.
```

Comment:
Again, I think this wording is misleading. We haven't actually changed any "defaults". We have only changed the suggested value in our `cluster.yaml` example and the helm chart.